internal/server/data/query.go

```diff
@@ -103,12 +122,19 @@ func update(tx WriteTxn, item Updatable) error {
 	query.B(item.Table())
 	query.B("SET")
 	query.B(columnsForUpdate(item), item.Values()...)
-	query.B("WHERE deleted_at is null AND id = ?;", item.Primary())
+	query.B("WHERE deleted_at is null")
+	query.B("AND id = ?", item.Primary())
+	query.B("AND organization_id = ?;", tx.OrganizationID())
```

Comment:
This is the same fix for the new SQL query approach.

Reply:
doesn't this need a check to see if the model has an org id?

Reply:
Ah, yes it does. I guess the tables that use it right now all have an OrgID. I'll add that check

Reply:
Done!